### ISSUE TITLE
Question for settings.php

### ISSUE THREAD

**Issue by robertjensen** [CINF/cinfdata#3]
Anybody remember the intended purpose of this line?
https://github.com/CINF/cinfdata/blob/91eb29f8959df03499ff322a9af94fc63ea672bc/sym-files2/graphsettings.php#L80

**Comment by KennethNielsen** [comment]
No, and the git blame isn't particularly useful: "Initial commit"

https://github.com/CINF/cinfdata/blame/91eb29f8959df03499ff322a9af94fc63ea672bc/sym-files2/graphsettings.php#L80

It looks like some sort of transformation

**Comment by KennethNielsen** [comment]
Searching for uses yield this: https://github.com/CINF/cinfdata/search?q=plot_settings

where we can see that e.g. one of the callers is passing in a "xscale" as the second argument, which gets turned into $params

**Comment by KennethNielsen** [comment]
which is being generated by the line above, via `date_xscale` which is here: https://github.com/CINF/cinfdata/blob/91eb29f8959df03499ff322a9af94fc63ea672bc/common_functions_v2.php#L63 ... ahhhh yes, now I remember.

This is the line that does the trick of formatting user provided parameters into the settings to make it possible to write `{param_name}` type variables in the settings XML file. It's probably a huge security whole BTW, but 🤷🏻 

so, if params is an array, like it is is there is a date_scale, the the array has `from` and `to` values and you then replace all occurrences in `{from}` and `{to}` in the settings XML file with those values.

Wow that was a trip down memory lane. I remember being very proud when I implemented that, and apparently a little too arrogant to include a comment. Better if I was the only one who knew 🙄
